iam-login-service/src/main/webapp/resources/iam/apps/dashboard-app/components/scopes/scopeslist/scopes.scopeslist.component.html

```diff
@@ -0,0 +1,67 @@
+<!--
+
+    Copyright (c) Istituto Nazionale di Fisica Nucleare (INFN). 2016-2019
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<!-- Current Active System Scopes Box -->
+<div class="box box-primary" id="access-tokens-list">
+    <div class="box-header with-border">
+        <div class="well well-small">
+            <button class="btn btn-small refresh-table"><i class="fa fa-refresh"></i> <span data-i18n="common.refresh">Refresh</span></button> &nbsp; 
+            <button class="btn btn-success" id="add-scope-btn" ng-click="$ctrl.openAddScopeDialog()"><i
+                class="fa fa-plus"></i> New Scope</button>
+        </div>
+        <table id="scope-table" class="table table-hover table-striped">
+            <thead>
+                <tr>
+                    <th></th>
+                    <th>Scope</th>
+                    <th></th>
+                    <th></th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr ng-repeat="scope in $ctrl.scopes">
+                    <td class="align-middle" ng-if="scope.defaultScope == true">
+                        <i class="fa fa-flag" title="This Scope is automatically assigned to newly registered clients."></i>
+                    </td>
+                    <td class="align-middle" ng-if="scope.restricted == true">
```

Comment:
move the ng-if  into "i" element and keep one and only column. (otherwise you have the bug showed in another comment)